dist/jexcel.js

```diff
@@ -0,0 +1,12891 @@
+    }
+
+    /**
+     * Get label
+     * 
+     * @param object cell
+     * @return string value
+     */
+    obj.getLabel = function(cell) {
+        // Convert in case name is excel liked ex. A10, BB92
+        cell = jexcel.getIdFromColumnName(cell, true);
+        var x = cell[0];
+        var y = cell[1];
+
+        return obj.records[y][x].innerHTML;
+    }
+
+    /**
+     * Get the value from a cell
+     * 
+     * @param object cell
+     * @return string value
+     */
+    obj.getValue = function(cell, processedValue) {
+        if (typeof(cell) == 'object') {
+            var x = cell.getAttribute('data-x');
+            var y = cell.getAttribute('data-y');
+        } else {
+            cell = jexcel.getIdFromColumnName(cell, true);
+            var x = cell[0];
+            var y = cell[1];
+        }
+
+        var value = null;
+
+        if (x != null && y != null) {
+            if (obj.records[y] && obj.records[y][x] && (processedValue || obj.options.copyCompatibility == true)) {
+                value = obj.records[y][x].innerHTML;
+            } else {
+                if (obj.options.data[y] && obj.options.data[y][x] != 'undefined') {
+                    value = obj.options.data[y][x];
+                }
+            }
+        }
+
+        return value;
+    }
+
+    /**
+     * Get the value from a coords
+     * 
+     * @param int x
+     * @param int y
+     * @return string value
+     */
+    obj.getValueFromCoords = function(x, y, processedValue) {
+        var value = null;
+
+        if (x != null && y != null) {
+            if ((obj.records[y] && obj.records[y][x]) && processedValue || obj.options.copyCompatibility == true) {
+                value = obj.records[y][x].innerHTML;
+            } else {
+                if (obj.options.data[y] && obj.options.data[y][x] != 'undefined') {
+                    value = obj.options.data[y][x];
+                }
+            }
+        }
+
+        return value;
+    }
+
+    /**
+     * Set a cell value
+     * 
+     * @param mixed cell destination cell
+     * @param string value value
+     * @return void
+     */
+    obj.setValue = function(cell, value, force) {
+        var records = [];
+
+        if (typeof(cell) == 'string') {
+            var columnId = jexcel.getIdFromColumnName(cell, true);
+            var x = columnId[0];
+            var y = columnId[1];
+
+            // Update cell
+            records.push(obj.updateCell(x, y, value, force));
+
+            // Update all formulas in the chain
+            obj.updateFormulaChain(x, y, records);
+        } else {
+            var x = null;
+            var y = null;
+            if (cell && cell.getAttribute) {
+                var x = cell.getAttribute('data-x');
+                var y = cell.getAttribute('data-y');
+            }
+
+            // Update cell
+            if (x != null && y != null) {
+                records.push(obj.updateCell(x, y, value, force));
+
+ 
```

Comment:
Add for column fixed with plugin tableHeadFixer.js
[tableHeadFixer.js.txt](https://github.com/paulhodel/jexcel/files/3767250/tableHeadFixer.js.txt)



```
var correction = 0;
        for(var ite_col in obj.records[0]) {
            if(obj.records[0][ite_col].style.position=="relative") {
                correction = Math.max(correction,obj.records[0][ite_col].getBoundingClientRect().left+obj.records[0][ite_col].getBoundingClientRect().width);
            } else {
                break;
            }
        }
        if(x2<correction) {
            x = x-correction-obj.content.scrollLeft+parseInt(obj.records[0][ite_col].style.left);
        }
```